packages/babel-plugin/src/__tests__/module-traversal.test.tsx

```diff
@@ -435,4 +449,109 @@ describe('module traversal', () => {
     expect(result).toHaveLength(1);
     expect(result[0]).toInclude('compiled/packages/babel-plugin/src/__fixtures__/mixins/simple.js');
   });
+
+  describe('Namespace imports', () => {
+    it('should replace a member expression referencing a default export', () => {
+      const result = transform(`
+        import '@compiled/react';
+        import React from 'react';
+        import * as objects from '../__fixtures__/mixins/objects';
+
+        <div css={{ color: objects.default.primary }} />
+      `);
+
+      expect(result).toInclude('{color:blue}');
+    });
+
+    it('should replace a member expression referencing a named variable export', () => {
+      const result = transform(`
+        import '@compiled/react';
+        import React from 'react';
+        import * as objects from '../__fixtures__/mixins/objects';
+
+        <div css={{ color: objects.colors.primary }} />
+      `);
+
+      expect(result).toInclude('{color:red}');
+    });
+
+    it('should replace a member expression referencing an export specifier', () => {
+      const result = transform(`
+        import '@compiled/react';
+        import React from 'react';
+        import * as objects from '../__fixtures__/mixins/objects';
+
+        <div css={{ color: objects.danger }} />
+      `);
+
+      expect(result).toInclude('{color:blue}');
+    });
+
+    it('should inline css from an object spread', () => {
```

Comment:
Should this include `referencing ...` like the other examples?

Reply:
I left it out of the test name as I wanted to just test spreading still works with namespaces. The previous 2 tests assert that export specifiers and variable declarations are correctly chained to the namespace. But I can add it to be more specific.